src/ngMock/angular-mocks.js

```diff
@@ -2600,7 +2602,17 @@ if (window.jasmine || window.mocha) {
     currentSpec = null;
 
     if (injector) {
-      injector.get('$rootElement').off();
+      var cleanUpElems = [originalRootElement];
+      var $rootElement = injector.get('$rootElement');
+      if ($rootElement !== originalRootElement) cleanUpElems.push($rootElement);
+      cleanUpElems.forEach(function(elem) {
+        // The `$rootElement` might not have been created or
+        // a mocked `$rootElement` might not have the standard methods
+        if (elem && elem.off) elem.off();
+        if (elem && elem.removeData) elem.removeData();
+      });
+      if (window.jQuery) window.jQuery.cleanData(cleanUpElems);
```

Comment:
The `jqLite` version of `cleanData` was added [fairly recently](https://github.com/angular/angular.js/commit/96288d02d316ff1508b2e0673f34291f9abd1c7f#diff-8e0d9e5c18fe4f1050484080525d4367R192)...